packages/account-abstraction-auth/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@safe-global/account-abstraction-auth",
+  "version": "0.1.0",
+  "description": "Safe Account Abstraction authentication library with web2 logins",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "scripts": {
+    "format:check": "prettier --check \"*/**/*.{js,json,md,ts}\"",
+    "format": "prettier --write */**/*.{js,json,md,ts}",
+    "unbuild": "rimraf dist",
+    "build": "rm -rf dist && tsc",
+    "test": "echo \"Error: no test specified\"",
+    "prepare": "yarn build"
+  },
+  "dependencies": {
+    "@safe-global/safe-core-sdk": "^3.3.1",
+    "@safe-global/safe-core-sdk-types": "^1.9.0",
```

Comment:
They should be removed as they are not used
```suggestion
```

Reply:
ok, yep